be/src/storage/compaction_scheduler.cpp

```diff
@@ -120,13 +120,13 @@ bool CompactionScheduler::_can_do_compaction_task(Tablet* tablet, CompactionTask
                       << ", running num:" << num;
             return false;
         }
-        last_failure_ms = tablet->last_base_compaction_failure_time();
+        last_failure_ts = tablet->last_base_compaction_failure_time();
     }
-    int64_t now_ms = UnixMillis();
-    if (now_ms - last_failure_ms <= config::min_compaction_failure_interval_sec * 1000) {
-        LOG(INFO) << "Too often to schedule compaction, skip it."
+    if (UnixMillis() - last_failure_ts <= config::min_compaction_failure_interval_sec * 1000) {
```

Comment:
UnixMillis() returns milliseconds, why did you rename _ms to _ts?

Reply:
ts short for timestamp